src/components/Pay2WinRow.vue

```diff
@@ -0,0 +1,89 @@
+<template>
```

Comment:
If you do end up keeping these new components, let's add either:
* A folder named `alchemy` to move these all into
* Prefix each component with `Alchemy`